transmission/event.go

```diff
@@ -11,6 +11,12 @@ import (
 	"github.com/vmihailenco/msgpack/v4"
 )
 
+type WireEvent struct {
+	Data       MarshallableMap `json:"data" msgpack:"data"`
+	SampleRate uint            `json:"samplerate,omitempty" msgpack:"samplerate,omitempty"`
+	Timestamp  *time.Time      `json:"time,omitempty" msgpack:"time,omitempty"`
```

Comment:
I kept the original order here, for compatibility with the tests, but ideally I'd expected to define and render in the opposite order:
- Timestamp
- SampleRate
- Data